lerna-package-lock.json

```diff
@@ -2047,6 +2047,13 @@
 						"@fluidframework/common-definitions": "^0.20.0",
 						"@fluidframework/core-interfaces": "^0.41.0",
 						"@fluidframework/protocol-definitions": "^0.1026.0"
+					},
```

Comment:
Are these changes related to the code changes in this PR?

Reply:
no. i don't know why the show up

Reply:
probably just semver and finding a new version